music21/roman.py

```diff
@@ -2094,34 +2118,34 @@ def changeFrontAlteration(intV, alter):
 
         if minorDefault == Minor67Default.FLAT:
             # default of flat does not need anything.
-            return
+            return workingFigure
 
         normallyRaised = self.impliedQuality in ('minor', 'diminished', 'half-diminished')
 
         if minorDefault == Minor67Default.SHARP:
-            sharpen()
+            return sharpen(workingFigure)
         elif minorDefault == Minor67Default.QUALITY:
             if not normallyRaised:
-                return
+                return workingFigure
             else:
-                sharpen()
-                return
+                return sharpen(workingFigure)
         else:  # CAUTIONARY
             if not self.frontAlterationAccidental or self.frontAlterationAccidental.alter == 0:
                 # same as QUALITY in this case
                 if not normallyRaised:
-                    return
+                    return workingFigure
                 else:
-                    sharpen()
+                    return sharpen(workingFigure)
```

Comment:
This is the only change to how early the function might exit. I overlooked at first that I was altering the control flow, but when I looked closer, upon exiting this branch,`frontAlter` below will be guaranteed to be 1, and `normallyRaised` is guaranteed to be `True`, so we will enter the branch that immediately returns without recalling `sharpen()`, so I don't see harm in returning here.

Reply:
:-)